yaml2ics.py

```diff
@@ -146,7 +147,7 @@ def events_to_calendar(events: list) -> str:
     return cal
 
 
-def files_to_events(files: list) -> (ics.Calendar, str):
+def files_to_events(files: list) -> (ics.Calendar, str, bool):
```

Comment:
Can we add some doc about that somewhere? The other parameters/return values were sort of ok to guess but this is less so to me.